templates/squad_v2/templates.yaml

```diff
@@ -0,0 +1,224 @@
+dataset: squad_v2
+templates:
+  17b83a3f-f748-42e6-9cdf-b2951dd8299d: !Template
+    id: 17b83a3f-f748-42e6-9cdf-b2951dd8299d
+    jinja: '{% set seq = [
+
+      ''Answer the question depending on the context.'',
+
+      ''What is the answer?'',
+
+      ] %}
+
+      {% if answers.text == [] %}
+
+      {{ seq | random }}
+
+      Context: {{context}};
+
+      Question: {{question}};
+
+      Answer: |||
+
+      Answer not in context
+
+      {% else %}
+
+      {{ seq | random }}
+
+      Context: {{context}};
+
+      Question: {{question}};
+
+      Answer: |||
+
+      {{answers.text[0]}}
+
+      {% endif %}'
```

Comment:
For easier read, let's factorize this: the prompt is always the same so let's move it outside of the "label" and keep the if condition `{% if answers.text == [] %}` only for the label part -> you won't have two `|||` which you almost never need